ceph.spec.in

```diff
@@ -231,7 +231,7 @@ BuildRequires:  openldap2-devel
 BuildRequires:  krb5
 BuildRequires:  krb5-devel
 BuildRequires:  cunit-devel
-BuildRequires:	python%{_python_buildid}-base
+BuildRequires:	python%{_python_buildid}-setuptools
```

Comment:
could you help me understand this change?

Reply:
In SUSE, the line `BuildRequires: python%{_python_buildid}-base` is not needed at all because we already have `BuildRequires: python%{_python_buildid}` in the non-distro-conditional section (`python%{_python_buildid}-base` is just a subset of `python%{_python_buildid}-base`). In other words, this line was never needed.

As for `python%{_python_buildid}-setuptools`, before this commit it was being brought in as a build dep on SUSE by this conditional block:

```
%if 0%{?rhel}
BuildRequires:	python34-devel
BuildRequires:	python34-setuptools
BuildRequires:	python34-Cython
%else
BuildRequires:	python3-devel
BuildRequires:	python3-setuptools
BuildRequires:	python3-Cython
%endif
```

Since this conditional block is RHEL/Fedora-specific, this commit moves that block into the overall RHEL/Fedora conditional block.